nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
+summary: Instructions on how to upgrade NServiceBus.Azure Storage Persistence Version 6 to NServiceBus.Persistence.AzureStorage Version 1.
+reviewed: 2016-05-12
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/azure-storage-persistence
+- nservicebus/upgrades/asp-saga-deduplication
+---
+
+DANGER: Upgrades from NServiceBus.Azure v6.2.3 or lower will need to apply the [saga de-duplication patch](/nservicebus/upgrades/asp-saga-deduplication.md) before completing the remainder of these upgrade steps.
```

Comment:
DANGER is a little strong. Maybe **NOTE** or **WARNING**.

Do we have a warning icon?